paperless-ngx.json

```diff
@@ -0,0 +1,113 @@
+                    },
+                    "/export": {
+                        "description": "Choose a Share for exported documents.",
+                        "label": "Config paperless Export - Storage"
+                    },
+                    "/consume": {
+                        "description": "Choose a Share for importing documents.",
+                        "label": "Config paperless Import - Storage"
+                    }
+                },
+                "environment": {
+                    "PAPERLESS_REDIS": {
+                        "description": "Set to redis://paperless-broker:6379.Redis connection URL. Do not change unless you know what you are doing.",
+                        "value": "redis://paperless-broker:6379",
+                        "label": "Redis URL"
+                    },
+                    "PAPERLESS_DBHOST": {
+                        "description": "Set to paperless-db. Hostname of the PostgreSQL database container.",
+                        "value": "paperless-db",
+                        "label": "Database Host"
+                    },
+                    "PAPERLESS_TIME_ZONE": {
+                        "description": "Enter timezone",
+                        "value": "Europe/Berlin",
+                        "label": "Timezone"
+                    }
+                }
+            },
+            "paperless-broker": {
+                "image": "docker.io/library/redis",
+                "tag": "7",
+                "launch_order": 1,
+                "volumes": {
+                    "/data": {
+                        "description": "Choose a Share for Redis data.",
+                        "label": "Redis Data Share"
+                    }
+                }
+            },
+            "paperless-db": {
+                "image": "docker.io/library/postgres",
+                "tag": "16",
+                "launch_order": 2,
+                "volumes": {
+                    "/var/lib/postgresql/data": {
+                        "description": "Choose a Share for PostgreSQL database storage.",
+                        "label": "PostgreSQL Storage"
+                    }
+                },
+                "environment": {
+                    "POSTGRES_DB": {
+                        "description": "Enter the database name for Paperless.",
+                        "label": "Database Name",
+                        "index": 1
+                    },
+                    "POSTGRES_USER": {
+                        "description": "Enter the username for the PostgreSQL database.",
+                        "label": "Database Username",
+                        "index": 2
+                    },
+                    "POSTGRES_PASSWORD": {
+                        "description": "Enter the password for the PostgreSQL database user.",
+                        "label": "Database Password",
+                        "index": 3
+                    }
```

Comment:
```suggestion
```

Move the environment variables into the `opts` section, since these don't need to be changed from the defaults.